aiobotocore/config.py

```diff
@@ -51,12 +60,16 @@ def _validate_connector_args(connector_args):
                     raise ParamValidationError(
                         report=f'{k} value must be an int or None'
                     )
-            elif k == 'keepalive_timeout':
+            elif k in ('keepalive_timeout', 'write_timeout', 'pool_timeout'):
```

Comment:
When checking for membership, tuples need to be iterated, which is not as efficient as checking e.g. in a set. I'd suggest to create `TIMEOUT_ARGS = frozenset(('keepalive_timeout', 'write_timeout', 'pool_timeout'))` outside the loop that iterates `connector_args` and check against that.

Reply:
with a container of size 3 I really don't think this optimization matters, and a quick benchmark gave a speedup of ~10%. But sure